tests/test-pkcs11-tool-unwrap-wrap-test.sh

```diff
@@ -76,7 +76,59 @@ cmp plain_wrapped.key aes_plain_key >/dev/null 2>/dev/null
 assert $? "wrapped key after decipher does not match the original key"
 
 echo "======================================================="
-echo " RSA-PKCS Cleanup"
+echo " RSA-PKCS-OAEP Unwrap test"
+echo "======================================================="
+ID4="97"
+ID5="98"
+# Reusing AES key and RSA key for unwrap/wrap operation already generated with ID1 in previous RSA-PKCS Unwrap test
+# Cleanup previously generated output
+rm aes_wrapped_key generic_extracted_key aes_ciphertext_openssl.data aes_ciphertext_pkcs11.data wrapped.key plain_wrapped.key
+
+# wrap AES key with RSA OAEP mode 
+# (SHA1 deprecated by NIST, therefore to be replaced with SHA256 or higher, but SoftHSM only supports SHA1 hash with OAEP currently, so we are stuck with SHA1 for now. Known issue: https://github.com/softhsm/SoftHSMv2/issues/474 .)
```

Comment:
just a nit, it would be good to have this parametrized somehow so when we will support something else than SHA1, we would just change it in one variable instead of going through every command.

Reply:
OK I've added variables for the OAEP hash / MGF algorithms at the start of the test, and these variables are now used in all the commands so that SHA1 can be replaced easily with something else in the future.